helm/ovn-kubernetes/charts/ovnkube-node-dpu/templates/ovnkube-node-dpu.yaml

```diff
@@ -229,8 +229,6 @@ spec:
           value: {{ hasKey .Values.global "enableMultiExternalGateway" | ternary .Values.global.enableMultiExternalGateway false | quote }}
         - name: OVNKUBE_NODE_MODE
           value: "dpu"
-        - name: OVNKUBE_NODE_MGMT_PORT_NETDEV
```

Comment:
Why was this removed?

Reply:
Per https://github.com/ovn-org/ovn-kubernetes/commit/2200972bfb4d1d7554196c7bd98a5620a79817e0, this information will be derived by the DPU from the annotation exposed on the host side.